DashAI/front/src/api/experiment.ts

```diff
@@ -1,18 +1,30 @@
 import api from "./api";
 import type { IExperiment } from "../types/experiment";
 
+const endpointURL = "/v1/experiment";
```

Comment:
Perhaps it would be good to re-centralize this type of configuration variables in a `.env` file. What do you think?

Reply:
I think it would be similar when using the backend, with the difference that all the urls would be in a single file.

The problem, I think, is that the variables would need to be duplicated in both `.env.development` and `.env.production`. However, if we want to allow them to have different values in each environment, I think it would be a good flexibility improvement.

Reply:
Ok, let's put it in tasks for the future.